internal/operators/manager.go

```diff
@@ -219,6 +232,59 @@ func (mgr *Manager) createControllerManifest(ctx context.Context, cluster *commo
 	return nil
 }
 
+// Create a ConfigMap containing the operator custom manifests and add it to install manifests.
+// This is needed when the assisted-installer cannot access the API to retrieve the manifest file, e.g.
+// for the Agent-Based Installer
+func (mgr *Manager) createManifestConfigMap(ctx context.Context, cluster *common.Cluster, manifests *[]Manifest) error {
```

Comment:
the name of this func sounds a little bit too generic, I'd suggesting something more specific, ie `createOLMOperatorsConfigMap`

Reply:
Done.